app.py

```diff
@@ -1,19 +1,59 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests as Requests
```

Comment:
It was in a suggestion online so I did it and went with it since flask has request and I thought it might be confusing if theres a requests, so I just used Requests with a capital to distinguish